src/DynamoCore/Graph/Nodes/DummyNode.cs

```diff
@@ -320,29 +416,29 @@ internal string GetDescription()
         {
             if (NodeNature == Nature.Deprecated)
             {
-                if (string.IsNullOrEmpty(LegacyAssembly))
+                if (string.IsNullOrEmpty(FunctionName))
                 {
-                    const string format = "Node of type '{0}' is now deprecated";
-                    return string.Format(format, LegacyNodeName);
+                    const string format = "Node of type '{0}',from assembly '{1}', is now deprecated.";
+                    return string.Format(format, TypeName, LegacyAssembly);
                 }
                 else
                 {
-                    const string format = "Node of type '{0}' ({1}) is now deprecated";
-                    return string.Format(format, LegacyNodeName, LegacyAssembly);
+                    const string format = "Node '{0}' is now deprecated";
+                    return string.Format(format, FunctionName);
                 }
             }
 
             if (NodeNature == Nature.Unresolved)
             {
-                if (string.IsNullOrEmpty(LegacyAssembly))
+                if (string.IsNullOrEmpty(FunctionName))
                 {
-                    const string format = "Node of type '{0}' cannot be resolved";
-                    return string.Format(format, LegacyNodeName);
+                    const string format = "Node of type '{0}', from assembly '{1}', cannot be resolved.";
+                    return string.Format(format, TypeName, LegacyAssembly);
                 }
                 else
                 {
-                    const string format = "Node of type '{0}' ({1}) cannot be resolved";
-                    return string.Format(format, LegacyNodeName, LegacyAssembly);
```

Comment:
is `LegacyNodeName` used anywhere anymore? If not maybe you want to obsolete it?